SeaBattle/BaseShip.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace SeaBattle
+{
+    public class BaseShip
+    { 
+        public List<Deck> Decks { get; set; }
+        int Durability;

```

Comment:
why this is a private field? not a property for example?

Reply:
just read stackoverflow and changed to properties because it’s much more flexibles option.

Reply:
for example in setter for durability I can hande event when durability falls below 0 and the ship is sinking.